generator.yaml

```diff
@@ -19,6 +17,14 @@ resources:
       Code:
         compare:
           is_ignored: true
+      Layers:
+        set:
+          - method: Create
+            ignore: true
+      LayersReference:
```

Comment:
@A-Hilaly how about doing something like this?

```yaml
resources:
  Function:
    fields:
      LayerStatuses:
        is_read_only: true
        from:
          operation: DescribeFunction
          path: Layers
```

and keep `Spec.Layers` as a `[]*string` containing the ARNs for the Function's Layers?

Reply:
Changed to the suggested configuration. But `Spec.Layers` is still `[]*string` and `Spec.LayerStatuses` is `[]*Layer`